docs/docs/concepts.mdx

```diff
@@ -1101,6 +1104,83 @@ The following how-to guides are good practical resources for using function/tool
 
 For a full list of model providers that support tool calling, [see this table](/docs/integrations/chat/#advanced-features).
 
+### Few-shot prompting
```

Comment:
Should there be some information on _why_ few shot prompting is important?

Reply:
as in beyond the first sentence?